### ISSUE TITLE
Task Fine Tuning Parameters

### ISSUE THREAD

**Issue by antonyscerri** [dmis-lab/biobert#7]
Hi

Would you be able to provide your exact parameter setup for the individual task fine tuning. It is unclear from the paper what was used for the reported numbers. Also between v2 and v3 of the paper you mentioned trying different ranges (v2 mentions epochs but in v3 you switched to mentioning learning rates).

Thanks

Tony

**Comment by jhyuklee** [comment]
Hi, Tony

We found that the number of epochs quite vary a lot depending on the tasks, or datasets to reach good performance, so we couldn’t enumerate all the tested epochs. You have to find the best epoch for each dataset using validation sets.
It usually takes more epochs for NER (>50), and less epochs for RE and QA (<10).
For learning rates, 1e-5 is recommended for NER, and default settings normally work for RE and QA.

Thanks,
Jinhyuk

**Comment by antonyscerri** [comment]
Hi

Did you use any early stopping criteria, or did you go back and through multiple epoch checkpoints and look for the best results?

Thanks

Tony

**Comment by jhyuklee** [comment]
We saved multiple checkpoints and tested them to find the best results.
Find the best epoch, and re-train the model including the validation set (which was often doen in prev works).

**Comment by antonyscerri** [comment]
Ok thank you for clarifying that.

Tony
